builtin/providers/datadog/resource_datadog_monitor.go

```diff
@@ -393,16 +393,9 @@ func resourceDatadogMonitorDelete(d *schema.ResourceData, meta interface{}) erro
 	return nil
 }
 
-func resourceDatadogImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
I think I didn't quite understand importing and this should be changed back. I'm still somewhat new to terraform and the importing functionality was a bit confusing to me.